packages/react/src/table/Table.stories.tsx

```diff
@@ -183,3 +186,46 @@ export const Modular: ComponentStory<typeof Table> = (args) => (
 		</Table>
 	</TableWrapper>
 );
+
+export const WithHeading = () => {
+	return (
+		<Stack gap={1}>
+			<H1 id="heading">Applications</H1>
+			<Text id="description">
+				This list of active establishment registrations
+			</Text>
+			<TableWrapper>
+				<Table aria-labelledby="heading" aria-describedby="description">
```

Comment:
Note for later: Typescript must not be running correctly on story files, cause this should be failing. You have named the prop `ariaLabelledby` not `aria-labelledby`. 

Note: The prop should be renamed, this should stay the same 👍

Reply:
Update: This is expected of typescript